### DISCUSSION TITLE
Multistroke / Repeat Slow Modified Keypress

### DISCUSSION THREAD

**Discussion by Myrkskog** [sezanzeb/input-remapper#722] [Q&A]
Within the multistroke if_tap setup; I noticed if I define the first stroke/key press with a hold_keys(x) it will allow me to press and release sending (x) once, and if double pressed + held it will repeatedly send (x).

if_tap(
	if_tap(
		modify(Shift_L, hold_keys(Super_R, left)),
		hold_keys(Alt_L, Escape) <---------WHAT TO DO HERE?! I would like a wait/delay when sending Escape each time slowly
	),
	modify(Shift_L, hold_keys(Super_R, down))
)

This key combination Alt + Esc cycles active windows in OS. I am hoping to maintain Multistroke functionality.
Cycling happens rapidly and its a roulette to land on the active window I want. How to slow this down?

**Comment by jonasBoss** [comment]
do exactly what you said: [wait](https://github.com/sezanzeb/input-remapper/blob/main/readme/macros.md#wait)
```
if_tap(
    if_tap(
        modify(Shift_L, hold_keys(Super_R, left)),
        hold_keys(Alt_L, Escape).wait(500)
    ),
    modify(Shift_L, hold_keys(Super_R, down))
)
```

if that does not work you can try to replace the inner `hold_keys` with a `modify` macro like this:
```
if_tap(
    if_tap(
        modify(Shift_L, hold_keys(Super_R, left)),
        modify(Alt_L, hold(Escape).wait(500))
    ),
    modify(Shift_L, hold_keys(Super_R, down))
)
```

**Reply by jonasBoss** [reply]
see also: https://github.com/sezanzeb/input-remapper/discussions/272#discussioncomment-2056874

**Reply by Myrkskog** [reply]
Thank you Jonasboss.

I tried what you suggested and I did not get the results I'm expecting. I think it relates to my logic of what I need here. Given I have another work around not input remapper related, I will shelf it for now.

Thanks again.
